lab5/Production.h

```diff
@@ -0,0 +1,30 @@
+//
+// Created by radoo on 11.12.2022.
+//
+
+#ifndef LAB5_PRODUCTION_H
+#define LAB5_PRODUCTION_H
+
+#include <vector>
+#include <string>
+
+class Production {
+private:
+    std::string nonTerminal;
+    std::vector<std::string> resultList;
+
+public:
+    Production(const std::string &nonTerminal, const std::vector<std::string> &resultList);
+    Production();
+
+    const std::string &getNonTerminal() const;
+
+    void setNonTerminal(const std::string &nonTerminal);
```

Comment:
Setters are prob. not needed. I assume we could have the Production and Grammar be immutable